### PR TITLE
Bug 1768662: kubevirt-web-ui in vendor main and getVMTemplate fix

### PR DESCRIPTION
**Pull request by mareklibra**
Based on: https://github.com/openshift/console/pull/3314
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1768662

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
-      </Link>
-    </>
-  );
-};
+export const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ name, namespace, uid }) => (
+  <>
+    <ResourceIcon kind={TemplateModel.kind} />
+    <Link
+      to={`/k8s/ns/${namespace}/vmtemplates/${name}`}
+      title={uid}
```
Comment (spadgett) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: I would just remove this. Showing the UID on hover isn't really helpful. I know we do this other places, but we're getting away from this in console.

Comment (spadgett) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: > I would just remove this. Showing the UID on hover isn't really helpful.

Don't worry about it for this PR.

**Comment by spadgett** [comment]
Previous failure was https://bugzilla.redhat.com/show_bug.cgi?id=1765280

/retest

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx (hunk 1 above)
Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: ok, I will push a follow-up once this gets merged

**Comment by spadgett** [comment]
I opened #3357 for the previous test flake

/retest

**Comment by spadgett** [comment]
Previous failure was https://bugzilla.redhat.com/show_bug.cgi?id=1765280

/retest

**Comment by spadgett** [comment]
Previous failure was https://bugzilla.redhat.com/show_bug.cgi?id=1765280

/retest

**Pull request merged**